### PR TITLE
Added address rendering and improved duration handling

### PR DESCRIPTION
**Pull request by kiramarston**
Screenshot 2023-02-02 at 14 59 33

### CODE REVIEW

File: packages/frontend/src/components/PopUpWrapper.js [hunk 1]
```diff
@@ -11,32 +11,36 @@ export const PopUpWrapper = ({ details, setActiveProperty }) => {
             }}
         >
             <div>
+                <h1>
```
Comment (ChrisPalmerAND) [inline packages/frontend/src/components/PopUpWrapper.js]: can we remove the h1? it's too big

File: packages/frontend/src/components/PopUpWrapper.js [hunk 2]
```diff
@@ -11,32 +11,36 @@ export const PopUpWrapper = ({ details, setActiveProperty }) => {
...
+                    {` ${details.walking.distance} miles / ${details.walking.duration} (hours and minutes)`}
                 </p>
                 <p>
                     <FontAwesomeIcon icon="fa-solid fa-train-subway" /> Train:
-                    {` ${details.train} (HH:MM:SS)`}
+                    {` ${details.train} (hours and minutes)`}
                 </p>
                 <p>
                     <FontAwesomeIcon icon="fa-solid fa-train-subway" /> Nearest Train Station:
                     {` ${details.nearestTrainStation.address}`}
```
Comment (ChrisPalmerAND) [inline packages/frontend/src/components/PopUpWrapper.js]: can we include the nearest station postcode too?

File: packages/frontend/src/components/PopUpWrapper.js (hunk 1 above)
Comment (kiramarston) [reply packages/frontend/src/components/PopUpWrapper.js]: How big would we like it to be?

Comment (ChrisPalmerAND) [reply packages/frontend/src/components/PopUpWrapper.js]: same as the rest?

**Comment by kiramarston** [comment]
Screenshot 2023-02-02 at 15 30 51
Additional improvements (postcodes, h1 to a bold p tag, bedrooms)

**Pull request merged**